src/configuration.ts

```diff
@@ -182,7 +184,7 @@ export class Configuration {
      * @type {Object|Null}
      * @defaultvalue null
      */
-    this._givenConfiguration = isObject(givenConfig) ? givenConfig : {};
+    this._givenConfiguration = isObject(givenConfig) ? givenConfig! : {};
```

Comment:
Isn't this simpler?
```ts
this._givenConfiguration = givenConfig || {};
```

Reply:
Yes, but that type of change needs to occur throughout the entire library.  I would like to focus that changes to its own PR.